index.html

```diff
@@ -51,8 +58,10 @@ <h1 class="name">I'm Ifeanyi, Glad to see you!</h1>
     </section>
     <section class="section1">
       <div class="main-container">
+        <div class="grid-container">
           <div class="project-container">
               <div class="developer"><img src="images/Snapshoot Portfolio.png" alt=""></div>
+              <div class="project-wrapper span-1">
               <div class="project-review">
                   <label>Tonic</label>
               </div>
```

Comment:
**OPTIONAL** - There are still some little differences between your design and the figma template. I'd suggest you try to adjust the height of each of your project cards 🙏. Know that replicating the exact specifications gives confidence to recruiters in your ability to design the exact webpage they'll require 👍  

Your design (Please pay attention to the margin between the `see project` button and the bottom margin of the card):
![project card student](https://user-images.githubusercontent.com/73607512/192995703-f090bf7c-c9ba-40d6-8d5a-26664617c45a.PNG)

Figma Template :
![work card student](https://user-images.githubusercontent.com/73607512/192995947-7a0ff460-21b6-4a5f-91b2-1270ae30ad92.PNG)